nucypher/characters/control/serializers.py

```diff
@@ -16,6 +16,24 @@ class SerializerError(ValueError):
         """Invalid data for I/O serialization or deserialization"""
 
 
+class MessageHandlerMixin:
+
+    __message_kit_transport_encoder = b64encode
+    __message_kit_transport_decoder = b64decode
+
+    def set_message_encoder(self, encoder: Callable):
+        self.__message_kit_transport_encoder = encoder
+
+    def set_message_decoder(self, decoder: Callable):
+        self.__message_kit_transport_decoder = decoder
+
+    def encode(self, plaintext: bytes) -> str:
+        return MessageHandlerMixin.__message_kit_transport_encoder(plaintext).encode()
```

Comment:
It seems this method is unused. Shouldn't be it used in symmetry with `decode()`?

Reply:
I actually didn't do anything with that class, I just moved it from lower in the code so I could re-use it in AliceControlJSONSerializer as it was originally only used in BobControlJSONSerializer.

I make no claim to knowing if it is or how it is used.